backend/src/apiserver/common/config.go

```diff
@@ -36,6 +36,7 @@ const (
 	CaBundleSecretName                      string = "CABUNDLE_SECRET_NAME"
 	RequireNamespaceForPipelines            string = "REQUIRE_NAMESPACE_FOR_PIPELINES"
 	CompiledPipelineSpecPatch               string = "COMPILED_PIPELINE_SPEC_PATCH"
+	DefaultPodNamespace                     string = "kubeflow"
```

Comment:
Just for consistency, I think this should go in `backend/src/apiserver/common/const.go`